numpy/lib/function_base.py

```diff
@@ -1622,12 +1622,13 @@ def trim_zeros(filt, trim='fb'):
     return filt[first:last]
 
 
-@deprecate
 def unique(x):
     """
     This function is deprecated.  Use numpy.lib.arraysetops.unique()
     instead.
     """
+    # turned into an error 2018-11-14
```

Comment:
This is never done, nor do I see a reason for it. Just remove the whole function now.

Reply:
+1 for removing it  Attribute error is just about as informative. (And besides, I’m pretty sure nobody is using this anymore)

Reply:
Mac test fails on unreachable code test.

Reply:
> Mac test fails on unreachable code test

Makes sense I think -- the code block below the new raise obviously isn't reachable now. Looks like the consensus will just be to remove the whole thing anyway.

Reply:
adopted